src/ldsout.c

```diff
@@ -74,7 +74,8 @@ unsigned sysout_loader(const char *sysout_file_name, unsigned sys_size) {
   struct stat stat_buf; /* file stat buf */
 
   char errmsg[255];
-
+  int cfp = -1;
```

Comment:
Should this be `int` or `off_t`?

Reply:
Yes, it probably should be `off_t` -- I knew it was at most a 256 MB file (that limit is baked into the Lisp memory structure because of bitfield sizes that will not change).

I'll fix that.